siphon/tests/test_acis.py

```diff
@@ -0,0 +1,41 @@
+'''Test ACIS Web Services API Access'''
+from siphon.simplewebservice.acis import acis_request
+from siphon.testing import get_recorder
+
+recorder = get_recorder(__file__)
+
+@recorder.use_cassette('acis_request')
+def test_acis():
```

Comment:
Can you break this up into 4 individual tests, one for each request method?

Reply:
I certainly can